docs/apache-airflow-providers-amazon/secrets-backends/aws-ssm-parameter-store.rst

```diff
@@ -67,9 +67,10 @@ you would want to store your connection at ``/airflow/connections/smtp_default``
 Optionally you can supply a profile name to reference aws profile, e.g. defined in ``~/.aws/config``.
 
 The value of the SSM parameter must be the :ref:`connection URI representation <generating_connection_uri>`
-of the connection object.
+or in the :ref:`JSON Format <connection-serialization-json-example>` of the connection object.
 
-In some cases, URI's you will need stored in Secrets Manager may not be intuitive, for example when using HTTP / HTTPS or SPARK, you may need URI's that will look like this:
+In some cases, URI's you will need stored in AWS SSM Parameter Store may not be intuitive,
```

Comment:
Is it me or the sentence `In some cases, URI's you will need stored in AWS SSM Parameter Store may not be intuitive` does not make a lot of sense ?

Reply:
Good point to change it. If it confuse you than it might confuse others.
From my side I've just change "Secrets Manager" to "AWS SSM Parameter Store" and split long line.

Reply:
Nice. I would make it even simpler.

`URIs stored in AWS SSM Parameter Store may not be intuitive`

Reply:
I thought that initial author tried to highlight that Airflow connection URI not a specific connection URI.

E.g. user expected that HTTP `https://example.org/path` would work however in Airflow we should use `http://https%3A%2F%2Fexample.com%2Fpath`

There is also a link to example how to generate URI exists in current doc: https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#generating-connection-uri